README.md

```diff
@@ -25,14 +25,14 @@ The app launches a window displaying a map.
 2. In the import wizard, choose _Maven > Existing Maven Projects_, then click _Next_.
```

Comment:
Verified sample builds and runs in Eclipse when I changed `arcgis.version` value to 100.5.0 in `pom.xml`. Need to re-verify at Update6 release, when 100.6.0 will be on bintray.